pontoon/base/static/js/main.js

```diff
@@ -525,23 +525,36 @@ $(function() {
         .find('section.' + section).show();
   });
 
-  // Toggle quality checks
-  $('.check-box.quality-checks').click(function() {
+  // Toggle user profile attribute
+  $('.check-box').click(function() {
     var self = $(this);
     Pontoon.startLoader();
 
     $.ajax({
-      url: '/quality-checks-switch/',
+      url: '/api/v1/user/' + Pontoon.user.email + '/',
       type: 'POST',
       data: {
-        csrfmiddlewaretoken: $('#server').data('csrf')
+        csrfmiddlewaretoken: $('#server').data('csrf'),
+        attribute: $(this).data('attribute')
```

Comment:
I don't think it's a blocker, but typically RESTful APIs have you write the value you want persisted rather than toggling the value, so that you don't have to read and then write the preference's value in order to set it correctly. Other possibilities include editing the preference in a different tab de-syncing the translate tab's UI from what's stored in the backend.

I'm totally down with shipping this and dealing with that later, though.